mule-extensions-api/src/main/java/org/mule/runtime/extension/api/introspection/ExtensionModel.java

```diff
@@ -92,8 +92,16 @@
   Optional<ConfigurationModel> getConfigurationModel(String name);
 
   /**
-   * Returns the {@link OperationModel}s which will be available to every
-   * {@link ConfigurationModel} defined in {@code this} extension
+   * Returns a {@link List} of {@link OperationModel}s defined at
+   * the extension
+   * level.
+   * <p>
+   * When an operation is defined at this level, it means that
+   * such operation does not require nor accept a configuration.
+   * <p>
+   * Each operation is guaranteed to have a unique name which will not
+   * overlap with any {@link SourceModel} or {@link ConnectionProviderModel}
```

Comment:
it's actually at any level and I should change it here and in `HasOperationModels`. Good catch